app/app.go

```diff
@@ -660,6 +661,13 @@ func NewLinkApp(
 	app.SetAnteHandler(anteHandler)
 	app.SetEndBlocker(app.EndBlocker)
 
+	// upgrade
+	app.UpgradeKeeper.SetUpgradeHandler(
+		upgradeName,
+		func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			return app.mm.RunMigrations(ctx, app.configurator, fromVM)
+		})
+
```

Comment:
Is there any way to be able to check this upgrade as unittest?

Reply:
I could not find a simple way to implement the unit test in golang. I may introduce a test in shell script, but it would be ugly. Any suggestions?

Reply:
No..